### ISSUE TITLE
[HW] <MEGHA_GOYAL>

### ISSUE THREAD

**Issue by MeghaGoyal2806** [DataStax-Academy/Intro-to-Cassandra-for-Developers#88]
**Name:** MEGHA GOYAL
**Email:**  goyalmegha286@gmail.com
**Linkedin Profile:** www.linkedin.com/in/megha-goyal-5a21bb180

<Screenshots>
![workshop1](https://user-images.githubusercontent.com/78904024/118110295-debea300-b3ff-11eb-8d00-d92ca804aeab.png)
![workshop2](https://user-images.githubusercontent.com/78904024/118110310-e2eac080-b3ff-11eb-88e3-a616d9725566.png)
![workshop3](https://user-images.githubusercontent.com/78904024/118110358-f138dc80-b3ff-11eb-9a35-f15e7e5c06a4.png)
![workshop4](https://user-images.githubusercontent.com/78904024/118110367-f39b3680-b3ff-11eb-9a25-a1ea90240a33.png)
![workshop5](https://user-images.githubusercontent.com/78904024/118110377-f72ebd80-b3ff-11eb-9afc-b1ada25539a8.png)
![workshop6](https://user-images.githubusercontent.com/78904024/118110399-fac24480-b3ff-11eb-90ce-d17cccb9b1df.png)
![cassandra query language](https://user-images.githubusercontent.com/78904024/118110413-fdbd3500-b3ff-11eb-8a00-82035df849d7.png)
![data modelling](https://user-images.githubusercontent.com/78904024/118110432-00b82580-b400-11eb-94a4-5150ae3cdee4.png)

**Comment by SonicDMG** [comment]
Hi MEGHA. Thanks for submitting your homework. I see almost everything except for the Data modeling scenario. However, I noticed that you tagged one of your images "data modeling", but it's actually am image of the Astra UI. Maybe the wrong image? Please add the Data modeling screenshot. :)

**Comment by MeghaGoyal2806** [comment]
Hello David, In instruction they mentioned that put screenshot of congratulation page. so i send it. what changes are needed?

**Comment by MeghaGoyal2806** [comment]
![Screenshot from 2021-05-13 15-09-31](https://user-images.githubusercontent.com/78904024/118134837-fa39a600-b41f-11eb-949b-b25ac8ceb0d3.png)

**Comment by SonicDMG** [comment]
Hi there. There is also this:
2. Complete the workshop practice as described below and make the screenshot of the last step ("Execute CRUD"). 

So just send the screenshot of the completed CRUD step after the DELETE command.

**Comment by SonicDMG** [comment]
Ah, there it is. That's what I needed. Thank you Megha.

**Comment by MeghaGoyal2806** [comment]
And for this '2. Complete the workshop practice as described below and make the screenshot of the last step ("Execute CRUD").' can you please see workshop6 ss

**Comment by SonicDMG** [comment]
Yes, sorry I have that one. That was my mistake when I typed my response. You are all good now.

**Comment by MeghaGoyal2806** [comment]
thanks david . good to hear from your side

**Comment by SonicDMG** [comment]
🍻

**Comment by MeghaGoyal2806** [comment]
Hello david.. there is any specific date for the result of this?

**Comment by SonicDMG** [comment]
Hi Megha. I believe you should see something today.

**Comment by HadesArchitect** [comment]
Here we go! Good job Megha! https://api.badgr.io/public/assertions/d2_fzGJ0QlifqVEex9s0Gg?identity__email=goyalmegha286%40gmail.com
